Tests/UnitTests/Networking/HTTPClientTests.swift

```diff
@@ -281,33 +315,67 @@ class HTTPClientTests: XCTestCase {
             )
         }
 
-        self.client.perform(request, authHeaders: [:]) {
-            result.value = $0
+        self.client.perform(request, authHeaders: [:]) { (response: HTTPResponse<CustomResponse>.Result) in
+            result.value = response
         }
 
         expect(result.value).toEventuallyNot(beNil(), timeout: .seconds(1))
         expect(result.value).to(beFailure())
+
+        // TODO: custom error?
+        let error = try XCTUnwrap(result.value?.error as NSError?)
+        expect(error.domain) == RCPurchasesErrorCodeDomain
+        expect(error.code) == ErrorCode.apiEndpointBlockedError.rawValue
```

Comment:
This is a placeholder (this test fails). The last TODO to improve error handling will replace this with actual better errors.